### PR TITLE
PR changes

### LINKED ISSUE #35: Show Pending PRs and Pending Issues first
Requirements:-

1. On the mentor's profile, in the PR Verification Requests section
    a. Show "pending PR requests" first and then accepted/rejected ones
    b. Also sort the "pending requests" such that the oldest appears first. Hint: You may use time-since filter.
2. On mentor's profile, In Issue Assignment Requests section
    a. Show "pending issue" requests first and then accepted/rejected ones
    b. Also sort the "pending issues" such that the oldest appears first.

This issue is currently assigned @kunal2812

### PR DESCRIPTION
**Pull request by Shouvik11-b**
fixes #35 
pr_requests_for_mentor
assignment_requests_for_mentor

changed the template to first show pending pr request and pending issue

### CODE REVIEW

File: user_profile/views.py [hunk 1]
```diff
@@ -35,11 +35,11 @@ def profile(request, username):
             # TODO: ISSUE Fetch User's Avatar's URL from Github API and display it in profile
             pr_requests_by_student = PullRequest.objects.filter(contributor=user)
             assignment_requests_by_student = IssueAssignmentRequest.objects.filter(requester=user)
-            active_issues = ActiveIssue.objects.filter(contributor=user)
+            active_issues = ActiveIssue.objects.filter(contributor=user).order_by('-submitted_at')
```
Comment (hackko-20) [inline user_profile/views.py:38]: There is no submitted_at field in ActiveIssue model. Kindly check again.

**Comment by hackko-20** [comment]
@Shouvik11-b Check the code once and resolve the comments. Also, try to test the code locally.

**Comment by hackko-20** [comment]
@Shouvik11-b are you working on this PR?

**Comment by Shouvik11-b** [comment]
No sir

**Pull request closed without merging**